src/main/knn/KnnIndexer.java

```diff
@@ -218,6 +218,7 @@ public int createIndex() throws IOException, InterruptedException {
           log("Indexed %d documents with %d parent docs. now flush", childDocs, parentDocs);
         }
       }
+      elapsed = System.nanoTime() - start;
```

Comment:
Can we move this under the `IndexWriter.commit`?  Else we are not measuring the time to write recently indexed docs as new segments...

Reply:
Sure, I moved to post commit now